it/uk/gov/hmrc/agentsubscriptionfrontend/controllers/StartControllerISpec.scala

```diff
@@ -49,7 +49,7 @@ class StartControllerISpec extends BaseControllerISpec {
       status(result) shouldBe OK
       contentType(result) shouldBe Some("text/html")
       charset(result) shouldBe Some("utf-8")
-      bodyOf(result) should include("This isn't an agent account")
+      bodyOf(result) should include(htmlEscapedMessage("This isn't an agent account"))
```

Comment:
The argument to `htmlEscapedMessage` should be a message key, not a plain string. It happens to work with plain strings but this is accidental.

i.e. `htmlEscapedMessage("nonAgent.title")`